src/docs/user/references/CalendarServiceReference.adoc

```diff
@@ -26,6 +36,10 @@ The URL for the logined user is going to be shown in a popup window
 
 image::calendar_url.png[align="center"]
 
+By default, only the secured URL is provided, which user can use with Thunderbird, Apple Calendar application with credential. A private URL can be created and used for Google calendar or Outlook without credential, the private URL can be regenerated or deleted if it is compromised.
```

Comment:
credential instead of credential. It is more common (https://en.oxforddictionaries.com/definition/credential)